Modules/Scripted/Home/Home.py

```diff
@@ -193,6 +195,34 @@ def downloadSampleDataInFolder(source):
             spec.loader.exec_module(setup)
             setup.setup()
 
+        # Pre-fill input/output fields in module
+        currModule = slicer.util.selectedModule()
+        if currModule == 'DataImporter':
```

Comment:
I suggest to use the following to get the module name:

```
if currModule == slicer.moduleNames.DataImporter:
```

It will fail right away if a module name changes